docs/topics/modules/api-get-stack-analyses-response.adoc

```diff
@@ -0,0 +1,53 @@
                        "watchers": 1673                        },                        "latest_version": "3.4.2",                        "license_analysis": {                            "_message": "Some unknown licenses found",                            "_representative_licenses": null,                            "conflict_licenses": [],                            "outlier_licenses": [],                            "status": "Unknown",                            "synonyms": {                                "Apache 2.0": "apache 2.0",                                "EPL 1.0": "EPL 1.0",                                "MIT License": "mit"                            },                            "unknown_licenses": [                                "EPL 1.0"                            ]                        },                        "licenses": [                            "Apache 2.0",                            "EPL 1.0",                            "MIT License"                        ],                        "name": "io.vertx:vertx-web",                        "osio_user_count": [                            6                        ],                        "security": [],                        "version": "3.4.2"                    }                ],                "analyzed_dependencies_count": 2,                "dependencies": [                    {                        "package": "io.vertx:vertx-core",                        "topic_list": [],                        "version": "3.4.2"                    },                    {                        "package": "io.vertx:vertx-web",                        "topic_list": [],                        "version": "3.4.2"                    }                ],                "distinct_licenses": [                    "EPL 1.0",                    "Apache 2.0",                    "MIT License"                ],                "ecosystem": "maven",                "license_analysis": {                    "conflict_packages": [],                    "f8a_stack_licenses": [],                    "outlier_packages": [],                    "status": "Unknown",                    "unknown_licenses": {                        "component_conflict": [],                        "really_unknown": [                            {                                "license": "EPL 1.0",                                "package": "io.vertx:vertx-core"                            },                            {                                "license": "EPL 1.0",                                "package": "io.vertx:vertx-web"                            }                        ]                    }                },                "recommendation_ready": true,                "stack_license_conflict": true,                "total_licenses": 3,                "unknown_dependencies": [],                "unknown_dependencies_count": 0            }        }    ],    "started_at": "2017-10-20T05:09:35.182243",    "version": "v1"}
```

Comment:
See above.

Reply:
Also, such large piece of source code should be included as a separate file to make the module file easier to view.